crates/gpui/src/window.rs

```diff
@@ -693,6 +693,8 @@ impl Window {
                             .update(&mut cx, |_, cx| {
                                 cx.draw();
                                 cx.present();
+                                #[cfg(target_os = "linux")]
+                                cx.update_ime_position();
```

Comment:
Shouldn't this just be called when the selection changes? Why does it need to be called on each render?